mod-users-bl/mod-users-bl.postman_collection.json

```diff
@@ -2610,73 +2494,73 @@
 	],
 	"variable": [
 		{
-			"id": "2f4a2ade-c097-4698-9726-e303a047a5fb",
+			"id": "4739d3ba-5439-4ec6-8813-6607cffcb80d",
 			"key": "module",
 			"value": "mod-users-bl",
 			"type": "string"
 		},
 		{
-			"id": "f03a5c4d-4bae-497a-8dd3-c527415875da",
+			"id": "6b8314d8-19c7-4fab-b320-1745c5ee6d6e",
 			"key": "module_version",
 			"value": "v4.4.0",
 			"type": "string"
 		},
 		{
-			"id": "27bcbba6-1652-4d4d-aa14-88173aaa6287",
+			"id": "f46cfc64-1589-4cae-9cb6-28f3a2a6a95f",
 			"key": "raml-utils_version",
 			"value": "0fdbe83a6fed316f0fed1b857ebefacc3abf9a72",
 			"type": "string"
 		},
 		{
-			"id": "3f46e3d4-ac4e-4a26-92e7-85cf12376b7e",
+			"id": "635d8af2-781b-448a-91fd-d627e429c987",
 			"key": "retrieveSchemaFunction",
 			"value": "var retrieveSchema = function(module, version, schemaLoc, schemaName, cb) {\n    pm.sendRequest({\n        url: \"https://raw.githubusercontent.com/folio-org/\" + module + \"/\" + version + schemaLoc + schemaName,\n        method: 'GET',\n    }, function(err, res) {\n        if (err !== null) {\n            console.log(\"Schema retrieval error for: \" + schemaName + \" - \" + err);\n            cb(\"failed\", null);\n        } else {\n            if (res.code === 200) {\n                cb(null, res.json());\n            } else {\n                console.log(\"Schema retrieval falied for: \" + schemaName + \" - \"  + res.reason());\n            cb(\"failed\", null);\n            }\n        }\n    });\n};\n\nvar retrieveModuleSchema = (schemaName, cb) => {\n\tconst module = pm.variables.get(\"module\");\n\tconst version = pm.variables.get(\"module_version\");\n\tretrieveSchema(module, version, \"/ramls/\", schemaName, cb);\n};\n\nvar retrieveRAMLUtilsSchema = function(schemaName, cb) {\n\tconst version = pm.variables.get(\"raml-utils_version\");\n\tretrieveSchema(\"raml\", version, \"/schemas/\", schemaName, cb);\n};",
 			"type": "string"
 		},
 		{
-			"id": "cefcd0bc-4d2c-4258-85e5-9328847fd15e",
+			"id": "539f3e08-9b94-4fa1-8b49-aea22e082711",
 			"key": "module_schemas",
-			"value": "compositeUser.json credentials.json proxyfor.json userdata.json usergroup.json permissionUser.json",
+			"value": "compositeUser.json credentials.json proxyfor.json userdata.json usergroup.json permissionUser.json\nservicepointsexpandeduser.json\nservicepoint.json",
```

Comment:
I see newline characters got appended just for these two schemas in the list. I am unsure if that is problematic when we run the collection. Probably not, but can we keep the list consistent?